package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "twilio-cli",
   "version": "2.21.0",
-  "description": "unleash the power of Twilio from your command prompt",
+  "description": "Unleash the power of Twilio from your command prompt. Visit https://www.twilio.com/docs/twilio-cli/quickstart for documentation.",
```

Comment:
Could shorten to [https://twil.io/cli](https://twil.io/cli)

Reply:
done